packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6675,7 +6675,7 @@ Update the logo for an application.
 | applicationId   | `id` of an [app](#application-object)      | URL              | String     | TRUE     |
 | file            | File containing logo                       | Body             | File       | TRUE     |
 
-The file must be in PNG, JPG, or GIF format, and less than 1 MB in size. For best results use landscape orientation, a transparent background, and a minimum size of 420px by 120px to prevent upscaling.
+The file must be in PNG, JPG, SVG, or GIF format, and less than 1 MB in size. For best results, use an image with a transparent background and a square dimension of 200px by 200px to prevent upscaling.
```

Comment:
"Upscaling" is probably not the right word here since what you want to say is that the image could be expanded and the quality reduced. But I think most readers will understand what the sentence is trying to say so this is just a comment. No action is needed.